docs/automation-example.md

```diff
@@ -0,0 +1,69 @@
+Here are some highlights on how to configure this (through the UI and taken from [this guide](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-run-lambda-schedule.html)):
+- Create a [Lambda function](https://console.aws.amazon.com/lambda/) that will receive data from EventBridge (you can create an empty fuction and work on it later), the data varies according to the event, for example: a push image to ECR event will have thise format and data (taken from [this guide](https://docs.aws.amazon.com/AmazonECR/latest/userguide/ecr-eventbridge.html#:~:text=The%20following%20event%20is%20sent%20when%20each%20image%20push%20is%20completed.%20For%20more%20information%2C%20see%20Pushing%20a%20Docker%20image.)):
+```
+{
+  "account": "123456789012",
+  "detail": {
+    "action-type": "PUSH",
+    "image-digest": "sha256:f98d67af8e53a536502bfc600de3266556b06ed635a32d60aa7a5fe6d7e609d7",
+    "image-tag": "latest",
+    "repository-name": "ubuntu",
+    "result": "SUCCESS"
+  },
+  "detail-type": "ECR Image Action",
+  "id": "4f5ec4d5-4de4-7aad-a046-56d5cfe1df0e",
+  "region": "us-east-1",
+  "resources": [],
+  "source": "aws.ecr",
+  "time": "2019-08-06T00:58:09Z",
+  "version": "0"
+}
+```
+- Go to your [Events page in AWS](https://console.aws.amazon.com/events/)
+- The event pattern will look somthing like this:
+```
+{
+  "source": ["aws.ecr"],
+  "detail-type": ["ECR Image Action"],
+  "detail": {
+    "action-type": ["PUSH"],
+    "result": ["SUCCESS"]
+  }
+}
+```
+
+## The Lambda function
+Your Lambda function will need to do the following in general:
+- Parse the new image's repository name and image tag from the data coming from the Event
+- Use that info to trigger am import job with the snyk-api-import tool
+
+To kick off an import, your Lambda function will need your SNYK_TOKEN, SNYK_ORG_ID and ORG_INTEGRATION_ID (or access to SSM with those parameters)
+
+### suggested architecture for the Lambda function
+- Get the "repository-name" and the "image-tag" from the event
+- Install or download the latest [snyk-api-import tool](https://github.com/snyk-tech-services/snyk-api-import/releases)
+- Create a targets file for the tool using your SNYK_ORG_ID, ORG_INTEGRATION_ID as explained [here](https://github.com/snyk-tech-services/snyk-api-import/blob/master/docs/import.md#1-create-the-import-projectsjson-file). The import file shsould have the neccessary fileds as needed by our [import endpoint](https://snyk.docs.apiary.io/#reference/import-projects/import-targets), for example - an import file for a new image will look something like this:
+```
+{
+   "targets":[
+        {
+         "integrationId":">ORG_INTEGRATION_ID>",
+         "orgId":"<SNYK_ORG_ID>",
+         "target":{
+            "name": "<REPOSITORY_NAME>:<IMAGE_TAG>"
+            }
+        }
+   ]
+}
+```
+- Set your SNYK_TOKEN and your SNYK_LOG_PATH as an enviroment variables
```

Comment:
wrap variables in backticks like this `SNYK_TOKEN` so they stand out

Reply:
Done